backend/src/api/Controllers/BlogController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using api.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Linq;
+
+namespace api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class BlogController : ControllerBase
+    {
+        private readonly ApiDbContext _context;
+
+        public BlogController(ApiDbContext context) 
+        {
+            _context = context;
+        }
+
+        [HttpGet]
+        public object Get()
+        {
+            return _context.Blogs.Where(b => b.Title.Contains("Title"))
```

Comment:
Using context directly in a controller is a terrible idea. I know that this is just to show that DB works but please take a look at https://docs.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design#the-repository-pattern and https://medium.com/net-core/repository-pattern-implementation-in-asp-net-core-21e01c6664d7